packaging/python_wheel/setup.py.in

```diff
@@ -14,8 +11,12 @@ def has_ext_modules(self):
 
 setup(
     name='brain',
-    version=VERSION,
-    description='brain',
+    version='${Brion_VERSION}',
```

Comment:
So, the build process for the Python package will generate the brain/version.py module automatically or do we still need to create it?